ocaml/xapi/xapi_globs.ml

```diff
@@ -1120,13 +1123,26 @@ let options_of_xapi_globs_spec =
   List.map
     (fun (name, ty) ->
       ( name
-      , (match ty with Float x -> Arg.Set_float x | Int x -> Arg.Set_int x)
+      , ( match ty with
+        | Float x ->
+            Arg.Set_float x
+        | Int x ->
+            Arg.Set_int x
+        | TimeSpanOfFloat x ->
+            Arg.Float
+              (fun y ->
+                let y' = Float.to_int (1000. *. y) in
+                x := Mtime.Span.(y' * ms)
+              )
+        )
       , (fun () ->
           match ty with
           | Float x ->
               string_of_float !x
           | Int x ->
               string_of_int !x
+          | TimeSpanOfFloat x ->
+              Fmt.to_to_string Mtime.Span.pp !x
```

Comment:
The only usage of the printer that I could find is to dump the config file to logs. It could be worth trying to get the original data (using span_to_s !x |> Option.get |> string_of_float). If the limits of conversion between spans and float seconds are simmetric, this shouldn't fail